Nim.cs

```diff
@@ -40,18 +84,43 @@ private static int GetChoice(int total)
             } while (choice < 1 || choice > 3);
             return choice;
         }
+        
+        private static int GetChoice100(int total)
```

Comment:
This function looks highly redundant with `GetChoice21`.  Can we refactor to eliminate the code duplication?